app/src/main/java/com/mapbox/vision/examples/activity/ar/ArNavigationActivity.kt

```diff
@@ -67,6 +68,7 @@ class ArNavigationActivity : AppCompatActivity(), RouteListener, ProgressChangeL
     private lateinit var routeFetcher: RouteFetcher
 
     private var lastRouteProgress: RouteProgress? = null
+    private var activeArFeature: ArFeature = ArFeature.Lane
```

Comment:
Does it mean only Lane is shown by default? Seems it's better to turn on everything to allow users to see new shiny Fence from the beginning

Reply:
I sticked to our old discussion stating that default parameters for `VisionArView` are `fenceVisiblity=false && laneVisibility=true`.
However I indeed like more an idea to turn everything on as default.
We'll return to that as soon as UI will be ready for the switch.

Reply:
@yunikkk - leaving `Lane` mode as default.

Reply:
@yunikkk - changed my mind - we will have enabled both `Lane` and `Fence` at start.